SPECS/bcc/bcc.spec

```diff
@@ -118,13 +129,21 @@ find %{buildroot}/usr/share/bcc/{tools,examples} -type f -exec \
 %{_datadir}/%{name}/man/*
 
 %changelog
-*   Fri Jun 05 2020 Suresh Babu Chalamalasetty <schalam@microsoft.com> 0.12.0-1
--   Update bcc version
-*   Tue Apr 21 2020 Eric Li <eli@microsoft.com> 0.10.0-4
--   Fix #Source0
-*   Tue Apr 07 2020 Paul Monson <paulmon@microsoft.com> 0.10.0-3
--   Add #Source0. License verified.
-*   Tue Sep 03 2019 Mateusz Malisz <mamalisz@microsoft.com> 0.10.0-2
--   Initial CBL-Mariner import from Photon (license: Apache2).
-*   Wed Jun 26 2019  Keerthana K <keerthanak@vmware.com> 0.10.0-1
--   Initial bcc package for PhotonOS.
+* Fri Sep 17 2021 Chris Co <chrco@microsoft.com> - 0.22.0-1
+- Update to 0.22.0
+- Add static library subpackage
```

Comment:
Why do we need the static libraries? We try to avoid them as much as possible, since they're a headache during updates.